pkg/sqlite/query.go

```diff
@@ -1168,3 +1168,25 @@ func (s *SQLQuerier) GetBundlePathIfExists(ctx context.Context, bundleName strin
 
 	return
 }
+
+func (s *SQLQuerier) GetBundleReplacesDepth(ctx context.Context, packageName, bundleName string) (replaceChain map[string]int64, err error) {
+	query := `SELECT replaces.operatorbundle_name, min(channel_entry.depth)
+			FROM channel_entry
+			inner JOIN channel_entry replaces ON channel_entry.replaces = replaces.entry_id
+			inner JOIN operatorbundle replacesbundle ON replaces.operatorbundle_name = replacesbundle.name
+			WHERE channel_entry.package_name = ? AND channel_entry.operatorbundle_name = ?
+			GROUP BY replaces.operatorbundle_name;`
```

Comment:
This query will be confused by bundles that appear in multiple channels with different skips/replaces.

Reply:
This is used to aggregate all the valid replaces entries by the least depth in all channels. From what I can see, it looks like this depth is unique for bundles added in replaces mode. 

semver mode requires ordering by versions instead, if there is ambiguity with this ordering, the exporter uses semver for choosing a replaces bundle

Reply:
I don't believe this stands to reason. For bundles with synthetic channel entries (those that have skips set at multiple levels or those inserted with --skip-patch mode), there will be duplicates of depth. As far as I know, depth isn't used to calculate the importance or truth of any of the replacements at all.